### PR TITLE
feat: project links

### PR DESCRIPTION
**Pull request by 1emu**
Closes #1818

image

image

### CODE REVIEW

File: src/components/Projects/ActionableLinksView.tsx [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+interface Props {
+  links: ProjectLink[]
+  projectId: string
+  isEditor: boolean
+}
+
+const NewLinkSchema: ZodSchema<Pick<ProjectLink, 'label' | 'url'>> = z.object({
+  label: z.string().min(1, 'Label is required').max(80),
+  url: z.string().min(0).max(200).url(),
```
Comment (ncomerci) [inline src/components/Projects/ActionableLinksView.tsx:30]: `min(0)` means optional?

**Review by ncomerci — APPROVED** [review]

File: src/components/Projects/ActionableLinksView.tsx (hunk 1 above)
Comment (1emu) [reply src/components/Projects/ActionableLinksView.tsx:30]: yes, there is an `optional` but it was not working with the `url` fn

File: src/clients/Governance.ts [hunk 2]
```diff
@@ -717,4 +718,20 @@ export class Governance extends API {
...
+  async createProjectLink(projectLink: ProjectLink) {
+    return await this.fetchApiResponse<ProjectLink>(`/projects/links/`, {
+      method: 'POST',
+      sign: true,
+      json: { project_link: projectLink },
+    })
+  }
+
+  async deleteProjectLink(projectLinkId: ProjectLink['id']) {
+    console.log('projectLinkId', projectLinkId)
```
Comment (andyesp) [inline src/clients/Governance.ts:731]: console.log!

File: src/components/Projects/ActionableLinksView.tsx [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+        <ProjectSidebarForm
+          initialValues={LINK_INITIAL_VALUES}
+          fields={NewLinkFields}
+          onSave={handleSave}
+          onCancel={handleCancel}
+          validationSchema={NewLinkSchema}
+          isFormDisabled={isFormDisabled}
+        />
+      )}
+      {!!error && <ErrorMessage label="Personnel Error" errorMessage={error} />}
```
Comment (andyesp) [inline src/components/Projects/ActionableLinksView.tsx:168]: "Link error" 👁️

**Pull request merged**